pkg/printer/print.go

```diff
@@ -249,7 +249,7 @@ func ProjectInfo(info models.ProjectInfo, jsonFlag bool) {
 }
 
 // Secrets print secrets
-func Secrets(secrets map[string]models.ComputedSecret, secretsToPrint []string, jsonFlag bool, plain bool, raw bool, copy bool) {
+func Secrets(secrets map[string]models.ComputedSecret, secretsToPrint []string, jsonFlag bool, plain bool, raw bool, copy bool, exitOnMissingSecret bool) {
```

Comment:
This `Secrets()` function lives in a package that's primarily intended to print values. It might be more appropriate for the caller to perform this check.

Reply:
👍  I should have done that from the get-go.